src/select.ts

```diff
@@ -43,7 +43,15 @@ function getEnclosingFormSelection(
     }
 }
 
-function selectForm(document = {}, selectionFn: Function, args?: any[]) {
+function selectForm(
+    document = {},
+    selectionFn: (
+        doc: vscode.TextDocument,
+        pos: vscode.Position,
+        ...topLevel: any
```

Comment:
```suggestion
        topLevel: boolean
```
Based on usage of `selectForm` and `getFormSelection` (which appears to be the only function passed for `selectionFn`), I think this change can be made.

Reply:
fixed here, took a couple more changes to make it all jive https://github.com/BetterThanTomorrow/calva/pull/1537/commits/9d049f67b64ad07eb8788a5615b74ddc7aef6706